protocols/enocean/org.eclipse.smarthome.protocols.enocean.eeps.basic/pom.xml

```diff
@@ -0,0 +1,111 @@
+<!--
+/*******************************************************************************
+ * Copyright (c) 2013, 2015 Orange.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Victor PERRON, Antonin CHAZALET, Andre BOTTARO.
+ *******************************************************************************/
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.smarthome.protocols.enocean</groupId>
+		<artifactId>org.eclipse.smarthome.protocols.enocean.parent</artifactId>
+		<version>0.8.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.eclipse.smarthome.protocols.enocean.eeps.basic</artifactId>
+
+	<packaging>bundle</packaging>
+
+	<name>${project.artifactId}</name>
+
+	<description>org.eclipse.smarthome.protocols.enocean.eeps.basic bundle</description>
+
+	<developers>
+		<developer>
+			<name>Victor PERRON</name>
+			<organization>Orange</organization>
+			<organizationUrl>orange.com</organizationUrl>
+			<email>victor.perron@orange.com</email>
+		</developer>
+		<developer>
+			<name>Antonin Chazalet</name>
+			<organization>Orange</organization>
+			<organizationUrl>orange.com</organizationUrl>
+			<email>antonin.chazalet@orange.com;antonin.chazalet@gmail.com</email>
+		</developer>
+		<developer>
+			<name>André BOTTARO</name>
+			<organization>Orange</organization>
+			<organizationUrl>orange.com</organizationUrl>
+			<email>andre.bottaro@orange.com</email>
+		</developer>
+	</developers>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-install-plugin</artifactId>
+				<version>2.3.1</version>
+				<executions>
+					<execution>
+						<id>install org.osgi.service.enocean maven artifact</id>
+						<phase>clean</phase>
+						<goals>
+							<goal>install-file</goal>
+						</goals>
+						<configuration>
+							<groupId>org.osgi.service</groupId>
+							<artifactId>enocean</artifactId>
+							<version>1.0.0</version>
+							<packaging>jar</packaging>
+							<file>${pom.basedir}/../maven_dependencies/org.osgi.service.enocean-1.0.0.jar</file>
+						</configuration>
+					</execution>
+					<execution>
+						<id>install comm.rxtx maven artifact</id>
+						<phase>clean</phase>
+						<goals>
+							<goal>install-file</goal>
+						</goals>
+						<configuration>
+							<groupId>nrjavaserial</groupId>
+							<artifactId>nrjavaserial</artifactId>
+							<version>3.9.3</version>
+							<packaging>jar</packaging>
+							<file>${pom.basedir}/../maven_dependencies/nrjavaserial-3.9.3.jar</file>
```

Comment:
generate a local maven repository with some jar files, that could be bundled
* https://github.com/maggu2810/maven-repo-generate

use a bundled maven repository
* https://github.com/maggu2810/maven-repo-bundle

You could use git submodule feature to host the local referenced maven repository at a place outside of eclipse (license reasons).
Or you could say the "user" have to checkout the local referenced maven repository for itself. If I understand your other comment correctly, this should be ok, because it is not part of the main bundles.